### PR TITLE
Support for React 16.0.0^ 

### PR DESCRIPTION
**Pull request by huynguyenh**
#5 

- Change from CreateClass to `extends Component`
- Use `prop-types` packge instead of `{ PropTypes } from react`  

cc @lorensr

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
   },
   "dependencies": {
     "formsy-react": "^0.18.1",
+    "prop-types": "^15.5.10",
     "lodash": "^4.14.1"
   },
   "peerDependencies": {
```
Comment (lorensr) [inline package.json:59]: I think this needs to change? And can you test by running storybook with react 16? Will require updating devDeps

**Comment by advance512** [comment]
Merge it in?

**Comment by nikulinsanya** [comment]
When are you going to merge it?  Just press the merge button!

**Comment by lorensr** [comment]
Would anyone like to complete the PR? See my comment above

**Comment by loanlaux** [comment]
Requesting for this to be closed. Solved in #10.

**Pull request closed without merging**